plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -237,7 +236,7 @@ func (c *MaxPDVolumeCountChecker) filterVolumes(volumes []v1.Volume, namespace s
 			if err != nil {
 				// if the PVC is not found, log the error and count the PV towards the PV limit
 				// generate a random volume ID since its required for de-dup
-				utilruntime.HandleError(fmt.Errorf("Unable to look up PVC info for %s/%s, assuming PVC matches predicate when counting limits: %v", namespace, pvcName, err))
+				glog.V(4).Infof("Unable to look up PVC info for %s/%s, assuming PVC matches predicate when counting limits: %v", namespace, pvcName, err)
```

Comment:
IMO it is not an error, it is basically useless to the reader. It's misleading because error implies something has broken but the error is ignored by the code. All this code does is it counts the # of gce PVs/PVCs used on a given node, the reader doesn't care if there was an error getting just one of those PVs/PVCs, the count will continue regardless, so this quickly becomes spam